expyfun/stimuli/_tracker.py

```diff
@@ -223,6 +237,10 @@ def respond(self, correct):
                     self._n_reversals += 1
                 if self._direction <= 0:
                     self._direction = 1
+        if self._repeat_limit == 'reversal':
```

Comment:
This should also do a step, right? So if you're at the floor, count it as a reversal and move up. Otherwise there's reversals with no actual change in direction.